packages/office-ui-fabric-react/src/components/__snapshots__/Dropdown.Custom.Example.tsx.shot

```diff
@@ -15,8 +17,8 @@ exports[`Component Examples renders Dropdown.Custom.Example.tsx correctly 1`] =
             box-shadow: none;
             box-sizing: border-box;
             color: #333333;
-            display: block;
```

Comment:
not sure if these are intentional... but I'm assuming it is?

Reply:
This is from: https://github.com/OfficeDev/office-ui-fabric-react/blob/546f0418fcdf56ad6d55af4141c2ac744cc473a6/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss#L96